console/src/main/scripts/package.json

```diff
@@ -32,6 +32,7 @@
     "gulp-uglify": "1.5.1",
     "gulp-ng-annotate": "1.1.0",
     "yargs": "3.31.0",
-    "gulp-if-else": "1.0.3"
+    "gulp-if-else": "1.0.3",
+    "gulp-protractor": "~2.1.0"
```

Comment:
Always use fixed versions. We have already been bitten by this. Trying to go back in time and getting a build to run, for example six months ago becomes almost impossible with non-specific versions that haven't even been tested together yet. The problem is much worse on the bower side of things than on the npm side of things just because of the conflict resolution policies and flat versus hierarchical library structures.